packages/dapp-connector/src/storage.ts

```diff
@@ -0,0 +1,37 @@
+import {App} from '@yoroi/types'
+
+export const connectionStorageMaker = (storage: App.Storage): Storage => {
+  const listAllConnections = async () => (await storage.getItem<DappConnection[]>('connections')) || []
+
+  const removeConnection = async (connection: DappConnection) => {
```

Comment:
use the correct convention
{
 connections: 
  {
    save..,
    read..,
    yadayada
  }
}